ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -2,83 +2,71 @@
+    path: C:\openjdk\OpenSSL-1.1.1e-x86_32
   register: openssl32_installed
   tags: openssl
 
 - name: Check if OpenSSL 64bit VS2013 installed
   win_stat:
-    path: C:\openjdk\OpenSSL-1.1.1d-x86_64
-  register: openssl64_installed
+    path: C:\openjdk\OpenSSL-1.1.1e-x86_64-VS2013
+  register: openssl64_vs2013_installed
   tags: openssl
 
 - name: Check if OpenSSL 64bit VS2017 installed
   win_stat:
-    path: C:\openjdk\OpenSSL-1.1.1d-x86_64.VS2017
-  register: openssl64_VS2017_installed
+    path: C:\openjdk\OpenSSL-1.1.1e-x86_64-VS2017
+  register: openssl64_vs2017_installed
   tags: openssl
 
-- name: Download OpenSSL-1.1.1d
+- name: Download OpenSSL-1.1.1e
   win_get_url:
-    url: https://www.openssl.org/source/openssl-1.1.1d.tar.gz
-    dest: C:\temp\openssl-1.1.1d.tar.gz
-    checksum: 1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2
+    url: https://www.openssl.org/source/openssl-1.1.1e.tar.gz
+    dest: C:\temp\openssl-1.1.1e.tar.gz
+    checksum: 694f61ac11cb51c9bf73f54e771ff6022b0327a43bbdfa1b2f19de1662a6dcbe
     checksum_algorithm: sha256
-  when: (not openssl32_installed.stat.exists) or (not openssl64_installed.stat.exists)
+  when: (not openssl32_installed.stat.exists) or (not openssl64_vs2013_installed.stat.exists) or (not openssl64_vs2017_installed.stat.exists)
   tags: openssl
 
-- name: Retrieve OpenSSL-1.1.1d 64-bit (VS2017)
-  win_get_url:
-    url: https://ci.adoptopenjdk.net/userContent/OpenSSL111win/OpenSSL111d-64bitVS2017.tgz
-    dest: C:\temp\openssl-1.1.1d-VS2017.tar.gz
-    checksum: 60ef36d98a369ad9086347b0cfb87816c472ac4b048e5aea178134e8ab82e0b1
-    checksum_algorithm: sha256
-  when: (not openssl64_VS2017_installed.stat.exists)
-  tags: openssl
-
-- name: Unpack OpenSSL-1.1.1d for installation
+- name: Unpack OpenSSL-1.1.1e for installation
   win_shell: |
     cd C:\temp
-    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1d.tar.gz
-    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1d.tar
-  when: (not openssl32_installed.stat.exists) or (not openssl64_installed.stat.exists)
+    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1e.tar.gz
+    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1e.tar
+  when: (not openssl32_installed.stat.exists) or (not openssl64_vs2013_installed.stat.exists) or (not openssl64_vs2017_installed.stat.exists)
   tags: openssl
 
-- name: Install OpenSSL-1.1.1d 32-bit
-  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vsvars32.bat && cd C:\temp\openssl-1.1.1d && perl C:\temp\openssl-1.1.1d\Configure VC-WIN32 --prefix=C:\openjdk\OpenSSL-1.1.1d-x86_32 && nmake install > C:\temp\openssl32.log && nmake -f makefile clean
+- name: Install OpenSSL-1.1.1e 32-bit (VS2013)
+  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vsvars32.bat && cd C:\temp\openssl-1.1.1e && perl C:\temp\openssl-1.1.1e\Configure VC-WIN32 --prefix=C:\openjdk\OpenSSL-1.1.1e-x86_32 && nmake install > C:\temp\openssl32.log && nmake -f makefile clean
```

Comment:
In the interests of consistence I think we should modify the 32-bit one to have the suffix too, just in case we do need the VS2017 version of that down the line at some point

Reply:
Commit in